x-pack/docs/en/security/troubleshooting.asciidoc

```diff
@@ -710,6 +710,25 @@ If you want your users to be able to use local credentials to authenticate to
 {kib} in addition to using the SAML realm for Single Sign-On, you must enable
 the `basic` `authProvider` in {kib}. The process is documented in the
 <<saml-kibana-basic, SAML Guide>>
+
+--
+
+. *Symptoms:*
++
+--
+No response ID values are being passed from {kib} to {es}
+
+....
+Caused by org.elasticsearch.ElasticsearchSecurityException: SAML content is in-response-to [_A1B2C3D4E5F6G8H9I0] but expected one of []
+....
+
+*Resolution:*
+
+Verify in your {kib} configuration that `xpack.security.sameSiteCookies` is not set to `Strict`.
```

Comment:
__note:__ it would be great if we could give more details. Something along these lines:

> This error indicates that {es} received a SAML response tied to a particular SAML request, but {kib} didn't explicitly specify ID of that request. This usually means that {kib} cannot find user session where it previously stored the SAML request ID.

> To resolve this issue, ensure that ... `xpack.security.sameSiteCookies` is not set to `Strict`. ..... (the rest of the text)

> If you serve multiple {kib} installations behind a load balancer make sure that you use [the same security configuration](https://www.elastic.co/guide/en/kibana/current/production.html#load-balancing-kibana) for all installations. 

What do you think?

Reply:
Definitely! Ill make the changes ✅

Reply:
Looks good to me now, thanks! Let's run by ES Security team